<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnmanedVichele</class>
 <widget class="QWidget" name="UnmanedVichele">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="LeftForward">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>750</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Forword</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RightForward">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>750</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Forword</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LeftBackward">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>790</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Backword</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RightBackward">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>790</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Backword</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>660</y>
     <width>361</width>
     <height>161</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_Video">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox
{
border:3px solid black;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="LeftSpeed">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>710</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>80</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="RightSpeed">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>710</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>80</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>680</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Left Wheels</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>680</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Right Wheels</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>620</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>IP address :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IPAddress">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>620</y>
     <width>191</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="StartOperation">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>620</y>
     <width>81</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ServoLeft">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>750</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ServoRight">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>750</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ServoDown">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>790</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ServoUp">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>710</y>
     <width>61</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>680</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo Motors</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ServoAngle">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>750</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>45</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>75</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>180</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="FlashLight">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>620</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Flash Light</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ObjectDetection">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>620</y>
     <width>131</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Object Detection</string>
   </property>
  </widget>
  <widget class="QComboBox" name="FlashIntensity">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>620</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>75</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
